ember/app/styles/components/_nav-bar.scss

```diff
@@ -1,6 +1,12 @@
 .nav-bar {
   border-bottom: solid 1px $global-border;
 
+  // prevent logo from shrinking when menu column width is smaller
+  // than logo width.
+  a.uk-logo {
+    flex: 0 0 auto;
+  }
+
```

Comment:
@Yelinz @derrabauke I need a second opinion here. Is that a sane thing to do?

Reply:
Yea looks sane. It's kinda same like `flex: none;` I think.